startup/01-installer.rules

```diff
@@ -0,0 +1 @@
+IMPORT{cmdline}="ANACONDA"
```

Comment:
Does there exist a more generic parameter that would disable RAID assembling?

Reply:
No it's the only flag available there (`65-md-incremental.rules`) - I guess that if there was a generic one they would likely not have bothered with a custom one anyway.

Reply:
Unconditionally setting that variable in our rule file should do the trick and avoid messing with boot commandlines, though.

Reply:
Where would this be done?

Reply:
Right, see updated patch